wgsl/index.bs

```diff
@@ -11662,6 +11668,43 @@ the rules in [[#floating-point-overflow]] apply.
 
 </table>
 
+#### Accuracy of AbstractFloat Expressions #### {#abstract-float-accuracy}
+
+The accuracy of an [=AbstractFloat=] operation is as follows:
+* A correct result is required when the corresponding [=f32=] operation requires a correct result.
+* A [=correctly rounded=] result is required when the corresponding [=f32=] operation requires a correctly rounded result.
+* Otherwise, the error of the corresponding [=f32=] operation is an absolute error, a relative error, an error inherited
+    from a potential implementation, or a combination of these.
+    In this case the error of the [=AbstractFloat=] is unbounded.
+    * However, the error of the [=AbstractFloat=] operation *should* be at most
+        the error of the corresponding operation in [=f32=], in absolute terms.
+    * This recommendation is meant to avoid surprises: the accuracy of
+        an expression should not be *reduced* when changing its type from f32 to AbstractFloat.
+    * The operation may be evaluated in a WebAssembly [[WASM-CORE-2]] or an ECMAScript [[ECMASCRIPT]]
+        environment, and those specifications do not specify error bounds on many
+        of the corresponding numeric computations.
+        For example, ECMAScript specifies many floating point operations as being
+        [[ECMASCRIPT#sec-terms-and-definitions-implementation-approximated|implementation-approximated]].
```

Comment:
Should we add a normative reference to go along with this?

Reply:
I originally had it as a normative reference. But the rule is fully specified here saying error is unbounded.
Strictly speaking this is a motivating explanation as to why the rule is the way it is.